docs/COMMUNICATION.rst

```diff
@@ -6,19 +6,18 @@ Feedback and Communication
 `Matrix chat <https://app.element.io/#/room/#plasmapy:matrix.org>`__
 --------------------------------------------------------------------
 
-The primary communication channel for PlasmaPy is our `Matrix room
-<https://app.element.io/#/room/#plasmapy:openastronomy.org>`__ (which
-is bridged to `Gitter <https://gitter.im/PlasmaPy/Lobby>`__).
+The primary communication channel, and the quickest way to ask a question and
+get a response, for PlasmaPy is our `Matrix room
+<https://app.element.io/#/room/#plasmapy:openastronomy.org>`__. It is is
+bridged (mirrored) to `Gitter <https://gitter.im/PlasmaPy/Lobby>`__.
 
-If you have any questions, the quickest way to get a response is to ask
-on our `Matrix <https://app.element.io/#/room/#plasmapy:matrix.org>`__/ channel.
-
-`Discourse group <https://plasmapy.discourse.group>`__
+`GitHub Discussions <https://github.com/PlasmaPy/PlasmaPy/discussions>`__
 ------------------------------------------------------
```

Comment:
```suggestion
`GitHub Discussions <https://github.com/PlasmaPy/PlasmaPy/discussions>`__
-------------------------------------------------------------------------
```

This is causing the doc build failure.

Reply:
Gah, sorry, i didn't check if the tests passed, just fired the pr off from terminal and forgot about it!